test/qcqpmodel.jl

```diff
@@ -182,6 +208,29 @@ context("With solver $(typeof(solver))") do
     @fact getValue(z) => roughly(1/sqrt(2), 1e-6)
     @fact getDual(eq) => roughly(sqrt(2), 1e-6)
 
+    # # SOC syntax version
+    # modN = Model(solver=solver)
+    # @defVar(modN, x >= 0)
+    # @defVar(modN, y)
+    # @defVar(modN, z)
+    # @setObjective(modN, Min, -y-z)
+    # @addConstraint(modN, eq, x <= 1)
+    # # @addConstraint(modN, y^2 + z^2 <= x^2)
+    # @addConstraint(modN, (1/4)*norm([4.0y,4.0z]) <= x)
+
+    # @fact solve(modN) => :Optimal
+    # @fact modN.objVal => roughly(-sqrt(2), 1e-6)
+    # @fact getValue(y) => roughly(1/sqrt(2), 1e-6)
+    # @fact getValue(z) => roughly(1/sqrt(2), 1e-6)
+    # @fact getDual(eq) => roughly(-sqrt(2), 1e-6)
```

Comment:
No, in particular the round-trip through JuMP form -> loadconicproblem! -> loadproblem! -> Gurobi -> JuMP is lossy and makes duals hard I think.